### ISSUE TITLE
UX/Accessibility issue - Dropdowns has no labels for screen readers

### ISSUE THREAD

**Issue by PereViki** [topcoder-platform/community-app#1521]
**Describe the bug**
When I try to navigate the site with a screen reader (Voice Over) the dropdown fields has no labels to be read out by the reader, only 'Combo box'. This make it impossible for the user in need to fill in the form.

**To Reproduce**
1. Turn on the screen reader (Voice Over)
2. Go to 'https://www.topcoder-dev.com/settings/profile'
2. Click on 'Basic Info'
3. Navigate to the dropdowns with the screen reader.

**Expected behavior**
Drodowns should have available labels for screen readers.

**Screenshots**
<img width="1280" alt="screenshot 2018-09-16 21 49 35" src="https://user-images.githubusercontent.com/35376779/45600282-6c9d2280-b9fa-11e8-93f3-33cd340b7ae1.png">

**Desktop: MacBook Pro**
 - OS: Mac OS High Sierra
 - Browser: Chrome
 - Version: 69.0.3497.92

@ThomasKranitsas
